dl/stubs.s

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2022 The Ebiten Authors
+
```

Comment:
Don't we need build tag restrictions?

```
//go:build darwin
// +build darwin
```

Reply:
This assembly file uses only a JMP which is found on all platforms so there really isn’t a need to. I can add one anyways since we are currently using this just for darwin

Reply:
Yeah, let's add the build tag for consistency.